pkg/utils/consts.go

```diff
@@ -1,5 +1,20 @@
 package utils
 
+import "github.com/prometheus/client_golang/prometheus"
+
 const (
-	HoursInMonth = 24.35 * 30 // 24.35 is the average amount of hours in a day over a year
+	HoursInMonth               = 24.35 * 30 // 24.35 is the average amount of hours in a day over a year
+	InstanceCPUCostSuffix      = "instance_cpu_usd_per_core_hour"
+	InstanceMemoryCostSuffix   = "instance_memory_usd_per_gib_hour"
+	InstanceTotalCostSuffix    = "instance_total_usd_per_hour"
+	PersistentVolumeCostSuffix = "persistent_volume_usd_per_hour"
 )
+
+func GenerateDesc(prefix, subsystem, suffix, description string, labels []string) *prometheus.Desc {
+	return prometheus.NewDesc(
+		prometheus.BuildFQName(prefix, subsystem, suffix),
+		description,
+		labels,
+		nil,
+	)
+}
```

Comment:
@ohyesgocool can you add some tests for this method please? Otherwise, we can pull this and test out your changes.

Reply:
Added in this [commit](https://github.com/grafana/cloudcost-exporter/pull/340/commits/d412b4185fe7720dfba9b6cbe7b04784ed7daa3e)